virt-v2v/cold/entrypoint.go

```diff
@@ -229,30 +214,40 @@ func executeVirtV2v(source string, args []string) (err error) {
 		return
 	}
 
-	if source == OVA {
-		scanner := bufio.NewScanner(teeErr)
-		const maxCapacity = 1024 * 1024
-		buf := make([]byte, 0, 64*1024)
-		scanner.Buffer(buf, maxCapacity)
-
-		for scanner.Scan() {
-			line := scanner.Bytes()
-			if match := UEFI_RE.FindSubmatch(line); match != nil {
-				fmt.Println("UEFI firmware detected")
-				firmware = "efi"
+	done := make(chan error, 1)
+	go func() {
```

Comment:
can we still get it with a TeeReader that reads from stdout of virt-v2v-monitor and writes to os.stdout?

Reply:
It's failing to set the tee reader since the os.stout already used for the monitor command.
`Error setting up stdout pipe: exec: Stdout already set`